src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,37 @@
+@import '../variables';
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+  width: 100%;
+  gap: 4px;
+
+  &__container {
+    display: flex;
+    margin-bottom: 24px;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url('/src/images/star.svg');
```

Comment:
The `background-image` path should be relative to the `index.scss` file. Ensure the path is correct so the image loads properly.